clients/js/test/transfer.test.ts

```diff
@@ -33,9 +35,8 @@ test('it can transfer a compressed NFT', async (t) => {
     creatorHash: hashMetadataCreators(metadata.creators),
     nonce: leafIndex,
     index: leafIndex,
-  })
-    .addRemainingAccounts([]) // <- Proof nodes would be added as remaining accounts.
-    .sendAndConfirm(umi);
+    proof: [],
```

Comment:
Can the default value be `[]` so you don't need to specify it?

Reply:
It already is! I just left them explicit in the tests so it's clear that this particular leaf don't need a proof. 🙂